lib/nexpose/global_settings.rb

```diff
@@ -115,5 +120,22 @@ def add_control_scanning_to_xml(xml, enabled)
                          'enabled' => enabled ? '1' : '0')
       end
     end
+
+    # Internal method for parsing XML for whether asset linking in enabled.
+    def parse_asset_linking_from_xml(xml)
+      enabled = false
```

Comment:
@kprzerwa-r7 @gschneider-r7 @sgreen-r7 @jhart-r7 I can't recall what the default value in Nexpose is, maybe it is actually `true`.